applications/picocdn-node-vm/templates/cdn-service.yaml

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2021 NetPico LAbs Pvt. Ltd.(picoNETS)
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "picocdn-node-vm.fullname" . }}-service
+spec:
+  type: {{ default "NodePort" .Values.cdnService.type }}
+  selector:
+    kubevirt.io/domain: {{ template "picocdn-node-vm.fullname" . }}
+  externalIPs:
+    192.168.1.100
+
+#  ports:
+#   -  port: {{ default "80" .Values.cdnService.port }}
+#{{- if .Values.cdnService.type }}
+#{{- if eq .Values.cdnService.type "NodePort" }}
+#      nodePort: {{ default "30007" .Values.cdnService.nodePort }}
+#{{- end }}
+#{{- else }}
+#      nodePort: {{ default "30007" .Values.cdnService.nodePort }}
+#{{- end }}
```

Comment:
Are these commented lines still needed?

Reply:
Try to cleanup unused and commented lines. But if they are required to be modified for different environments and kept for reference/example. it is ok, but put a comment for the same.

Reply:
These lines were a leftover block from the OpenNESS CDN application template, which we used as an initial baseline for our application. We have removed it, to have a cleaner and more compact file.